hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -223,6 +226,15 @@ public OzoneContainer(
             blockDeletingServiceTimeout, TimeUnit.MILLISECONDS,
             blockDeletingServiceWorkerSize, config);
 
+    Duration diskBalancerSvcInterval = conf.getObject(
+        DatanodeConfiguration.class).getDiskBalancerInterval();
+    Duration diskBalancerSvcTimeout = conf.getObject(
+        DatanodeConfiguration.class).getDiskBalancerTimeout();
+    diskBalancerService =
```

Comment:
diskBalancerService should be started and stopped according in OzoneManager start and stop function.

Reply:
Updated.